formatters/partials/speedline.html

```diff
@@ -0,0 +1,13 @@
+<style>
+  .speedline-measures {
+    font-size: 14px
+  }
+</style>
+
+<div>
+  <div class="speedline-measures">
+    <div>First Visual Change: <strong>{{this.first}}ms</strong></div>
+    <div>Last Visual Change: <strong>{{this.complete}}ms</strong></div>
+    <div>Recording duration: <strong>{{this.duration}}ms</strong></div>
```

Comment:
we can nuke recording duration from the return value even. doesn't need to be reported.